.gitattributes

```diff
@@ -15,3 +15,5 @@
 *.rc diff
 
 *.gcode linguist-detectable=false
+
+.vs-config linguist-language=json

```

Comment:
shouldn't this be .vsconfig, not vs-config?

Reply:
Fixed!